### ISSUE TITLE
how to get the dataset split？

### ISSUE THREAD

**Issue by wuxiaofei01** [GentlesJan/AffordPose#7]
Hello author, where can I get the training set, test set, and validation set of the dataset?

**Comment by GentlesJan** [comment]
Hi. Unfortunately, we have not provided the dataset's predefined training, test, and validation sets. However, you are welcome to divide the data according to your specific needs and use case.










Best regards,
Jan

At 2024-10-14 16:48:55, "wuxiaofei" ***@***.***> wrote:

Hello author, where can I get the training set, test set, and validation set of the dataset?
